### PR TITLE
br, sql-statements: document BACKUP, RESTORE and SHOW [BACKUPS|RESTORES]

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

- Add a doc for BR storages.
- Add docs for 3 SQL statements:

    - BACKUP
    - RESTORE
    - SHOW BACKUPS

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/2618
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review this PR. @TomShawn

File: sql-statements/sql-statement-backup.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+# BACKUP
+
+`BACKUP` 语句用于对 TiDB 集群执行分布式备份操作。
+
+`BACKUP` 语句使用的引擎与 [BR](/br/backup-and-restore-use-cases.md) 相同，但备份过程是由 TiDB 本身驱动，而非单独的 BR 工具。BR 工具的优势和警告也适用于 `BACKUP` 语句。
+
+执行 `BACKUP` 需要 `SUPER` 权限。此外，执行备份的 TiDB 节点和集群中的所有 TiKV 节点都必须有对目标存储的读或写权限。
+
+`BACKUP` 语句开始执行后将会被封锁，直到整个备份任务完成、失败或取消。因此，执行 `BACKUP` 时需要准备一个持久的连接。如需取消任务，可执行 [`KILL TIDB QUERY`](/sql-statements/sql-statement-kill.md) 语句。
```
Comment (TomShawn) [inline sql-statements/sql-statement-backup.md:15]: `封锁` sounds a bit weird😲. Is it what you mean @kennytm ?

File: sql-statements/sql-statement-backup.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+上述示例中，`test` 数据库被备份到本地，数据以 SST 文件的形式存储在分布于所有 TiDB 和 TiKV 节点的 `/mnt/backup/2020/04/` 目录中。
+
+输出结果的第一行描述如下：
+
+| 列名 | 描述 |
+| :-------- | :--------- |
+| `Destination` | 目标存储的 URL |
+| `Size` |  备份文件的总大小，单位为字节 |
+| `BackupTS` | 创建备份时的快照 TSO（用于[增量备份](#增量备份)） |
+| `Queue Time` | `BACKUP` 任务开始排队的 timestamp（当前时区） |
```
Comment (TomShawn) [inline sql-statements/sql-statement-backup.md:69]:
```suggestion
| `Queue Time` | `BACKUP` 任务开始排队的时间戳（当前时区） |
```

File: sql-statements/sql-statement-backup.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+| `Execution Time` | `BACKUP` 任务开始执行的 timestamp（当前时区） |
```
Comment (TomShawn) [inline sql-statements/sql-statement-backup.md:70]:
```suggestion
| `Execution Time` | `BACKUP` 任务开始执行的时间戳（当前时区） |
```

File: sql-statements/sql-statement-backup.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+### 备份集群
+
+{{< copyable "sql" >}}
+
+```sql
+BACKUP DATABASE * TO 'local:///mnt/backup/full/';
+```
+
+注意，备份中不包含系统表 (`mysql.*`, `INFORMATION_SCHEMA.*`, `PERFORMANCE_SCHEMA.*`, …)。
```
Comment (TomShawn) [inline sql-statements/sql-statement-backup.md:94]:
```suggestion
注意，备份中不包含系统表 (`mysql.*`、`INFORMATION_SCHEMA.*`、`PERFORMANCE_SCHEMA.*` 等)。
```

File: sql-statements/sql-statement-restore.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+title: RESTORE
+summary: TiDB 数据库中 RESTORE 的使用概况。
+category: reference
+---
+
+# RESTORE
+
+`RESTORE` 语句用于执行分布式恢复，把 [`BACKUP` 语句](/sql-statements/sql-statement-backup.md)生成的备份文件恢复到 TiDB 集群中。
+
+`RESTORE` 语句使用的引擎与 [BR](/br/backup-and-restore-use-cases.md) 相同，但恢复过程是由 TiDB 本身驱动，而非单独的 BR 工具。BR 工具的优势和警告也适用于 `BACKUP` 语句。需要注意的是，**`RESTORE` 语句目前不遵循 ACID 原则**。
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:11]:
```suggestion
`RESTORE` 语句使用的引擎与 [BR](/br/backup-and-restore-use-cases.md) 相同，但恢复过程是由 TiDB 本身驱动，而非单独的 BR 工具。BR 工具的优势和警告也适用于 `RESTORE` 语句。需要注意的是，**`RESTORE` 语句目前不遵循 ACID 原则**。
```

File: sql-statements/sql-statement-restore.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+执行 `RESTORE` 语句前，确保集群已满足以下要求：
+
+* 集群处于“下线”状态，当前的 TiDB 会话是唯一在访问要恢复的表的活跃 SQL 连接。
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:15]:
```suggestion
* 集群处于“下线”状态，当前的 TiDB 会话是唯一在访问待恢复表的活跃 SQL 连接。
```

File: sql-statements/sql-statement-restore.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+* 执行全量恢复时，确保即将恢复的表不存在于集群中，因为现有的数据可能被覆盖，从而导致数据与索引不一致。
+* 执行增量恢复时，表的状态应该与创建备份时 `LAST_BACKUP` 时间戳的状态完全一致。
+
+执行 `RESTORE` 需要 `SUPER` 权限。此外，执行恢复操作的 TiDB 节点和集群中的所有 TiKV 节点都必须有对目标存储的读权限。
+
+`RESTORE` 语句开始执行后将会被封锁，直到整个恢复任务完成、失败或取消。因此，执行 `RESTORE` 时需要准备一个持久的连接。如需取消任务，可执行 [`KILL TIDB QUERY`](/sql-statements/sql-statement-kill.md) 语句。
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:21]: ditto. `封锁`

File: sql-statements/sql-statement-restore.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+
+一次只能执行一个 `BACKUP` 和 `RESTORE` 任务。如果 TiDB server 上已经在执行一个 `BACKUP` 或 `RESTORE` 语句，新的 `RESTORE` 将等待前面所有的任务完成后再执行。
+
+`RESTORE` 只能在 "tikv" 存储引擎上使用，如果使用 "mocktikv" 存储引擎， `RESTORE` 操作会失败。
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:25]:
```suggestion
`RESTORE` 只能在 "tikv" 存储引擎上使用，如果使用 "mocktikv" 存储引擎，`RESTORE` 操作会失败。
```

File: sql-statements/sql-statement-restore.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+
+输出结果的第一行描述如下：
+
+| 列名 | 描述 |
+| :-------- | :--------- |
+| `Destination` | 读取的目标存储 URL |
+| `Size` |  备份文件的总大小，单位为字节 |
+| `BackupTS` | 不适用 |
+| `Queue Time` | `RESTORE` 任务开始排队的 timestamp（当前时区） |
+| `Execution Time` | `RESTORE` 任务开始执行的 timestamp（当前时区） |
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:74]:
```suggestion
| `Execution Time` | `RESTORE` 任务开始执行的时间戳（当前时区） |
```

File: sql-statements/sql-statement-restore.md [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+
+### 部分恢复
+
+你可以指定恢复部分数据库或表的数据。如果备份文件中缺失了某些数据库或表，缺失的部分将被忽略。此时，`RESTORE` 语句不进行任何操作即完成执行。
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:78]:
```suggestion
你可以指定恢复部分数据库或部分表数据。如果备份文件中缺失了某些数据库或表，缺失的部分将被忽略。此时，`RESTORE` 语句不进行任何操作即完成执行。
```

File: sql-statements/sql-statement-restore.md [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+上述示例中，所有数据均从本地的备份文件中恢复到集群中。`RESTORE` 从 SST 文件里读取数据，SST 文件存储在所有 TiDB 和 TiKV 节点的 `/mnt/backup/2020/04/` 目录下。
+
+输出结果的第一行描述如下：
+
+| 列名 | 描述 |
+| :-------- | :--------- |
+| `Destination` | 读取的目标存储 URL |
+| `Size` |  备份文件的总大小，单位为字节 |
+| `BackupTS` | 不适用 |
+| `Queue Time` | `RESTORE` 任务开始排队的 timestamp（当前时区） |
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:73]:
```suggestion
| `Queue Time` | `RESTORE` 任务开始排队的时间戳（当前时区） |
```

File: sql-statements/sql-statement-restore.md [hunk 12]
```diff
@@ -0,0 +1,161 @@
...
+
+![BRIETables](/media/sqlgram/BRIETables.png)
+
+**RestoreOption:**
+
+![RestoreOption](/media/sqlgram/RestoreOption.png)
+
+**Boolean:**
+
+![Boolean](/media/sqlgram/Boolean.png)
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:43]: This image looks different from the one in https://github.com/pingcap/docs/pull/2618. 
image

File: sql-statements/sql-statement-backup.md [hunk 13]
```diff
@@ -0,0 +1,186 @@
...
+
+## 语法图
+
+**BackupStmt:**
+
+![BackupStmt](/media/sqlgram/BackupStmt.png)
+
+**BRIETables:**
+
+![BRIETables](/media/sqlgram/BRIETables.png)
```
Comment (TomShawn) [inline sql-statements/sql-statement-backup.md:27]: This image looks different from the one in https://github.com/pingcap/docs/pull/2618
image

File: sql-statements/sql-statement-backup.md (hunk 1 above)
Comment (kennytm) [reply sql-statements/sql-statement-backup.md:15]: better use `加锁` or something like that.

Comment (ran-huang) [reply sql-statements/sql-statement-backup.md]: Updated. Thanks 😃

**Comment by ran-huang** [comment]
@TomShawn All comments addressed. PTAL, thanks~

File: br/backup-and-restore-storages.md [hunk 14]
```diff
@@ -0,0 +1,82 @@
+---
+title: BR 存储
+summary: 了解 BR 中所用存储服务的 URL 格式。
+category: reference
```
Comment (TomShawn) [inline br/backup-and-restore-storages.md:4]:
```suggestion
```

File: sql-statements/sql-statement-backup.md [hunk 15]
```diff
@@ -0,0 +1,186 @@
+---
+title: BACKUP
+summary: TiDB 数据库中 BACKUP 的使用概况。
+category: reference
```
Comment (TomShawn) [inline sql-statements/sql-statement-backup.md:4]:
```suggestion
```

File: sql-statements/sql-statement-restore.md [hunk 16]
```diff
@@ -0,0 +1,161 @@
+---
+title: RESTORE
+summary: TiDB 数据库中 RESTORE 的使用概况。
+category: reference
```
Comment (TomShawn) [inline sql-statements/sql-statement-restore.md:4]:
```suggestion
```

File: sql-statements/sql-statement-show-backups.md [hunk 17]
```diff
@@ -0,0 +1,99 @@
+---
+title: SHOW [BACKUPS|RESTORES]
+summary: TiDB 数据库中 SHOW [BACKUPS|RESTORES] 的使用概况。
+category: reference
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-backups.md:4]:
```suggestion
```

**Pull request merged**